WORKSPACE

```diff
@@ -1,4 +1,4 @@
-workspace(name = "bazel_gcc_toolchain")
+workspace(name = "aspect_bazel_gcc_toolchain")
```

Comment:
we generally have `aspect-build/rules_foo` -> `aspect_rules_foo`
so maybe drop the "bazel" from this?

Reply:
Addressed in [dc4bcea](https://github.com/aspect-build/gcc-toolchain/pull/38/commits/dc4bcea10d8ed74ccb21486716aaf9892bee4701).